.github/workflows/release.yaml

```diff
@@ -13,17 +13,27 @@ jobs:
 
   release:
     needs: check
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.runs-on }}
+    strategy:
+      fail-fast: false
+      matrix:
+        runs-on:
+          - [ubuntu-latest]
     steps:
       - name: Checkout
         uses: actions/checkout@v4
+        with:
+          submodules: true
+
       - name: Initialize lxd  # This should dropped once it's implemented on charming-actions itself. https://github.com/canonical/charming-actions/issues/140
-        uses: canonical/setup-lxd@v0.1.1
-      - name: Upload charm to charmhub
-        uses: canonical/charming-actions/upload-charm@2.4.0
+        # revision is latest main at time of writing; using because it contains a fix to https://github.com/canonical/setup-lxd/issues/19
+        uses: canonical/setup-lxd@2aa6f7caf7d1484298a64192f7f63a6684e648a4
+
+      - name: Pack and upload to charmhub
+        uses: canonical/charming-actions/upload-charm@2.6.2
```

Comment:
should we stick with 2.6.1 for now due to https://github.com/canonical/charming-actions/issues/148 ?

Reply:
We've been using 2.6.2 for several projects and haven't observed that bug. :)